x-pack/plugins/rollup/public/crud_app/sections/job_create/steps/step_histogram.js

```diff
@@ -0,0 +1,239 @@
+      truncateText: true,
+      sortable: true,
+    }];
+
+    const unselectedHistogramFields = histogramFields.filter(histogramField => {
+      return !fields.histogram.includes(histogramField);
+    });
+
+    return (
+      <Fragment>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <h3>
+                <FormattedMessage
+                  id="xpack.rollupJobs.create.stepHistogram.title"
+                  defaultMessage="Histogram (optional)"
+                />
+              </h3>
+            </EuiTitle>
+
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.rollupJobs.create.stepHistogram.description"
+                  defaultMessage={`
+                    Select the fields you want to bucket using numeric histgogram intervals.
+                  `}
+                />
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              size="s"
+              flush="right"
+              href={histogramDetailsUrl}
+              target="_blank"
+              iconType="help"
+            >
+              <FormattedMessage
+                id="xpack.rollupJobs.create.stepHistogram.readDocsButton.label"
+                defaultMessage="Read the docs"
+              />
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <FieldList
+          columns={columns}
+          fields={histogram}
+          onRemoveField={this.onRemoveField}
+        />
+
+        <EuiSpacer />
+
+        <FieldChooser
+          columns={columns}
+          label={(
+            <FormattedMessage
+              id="xpack.rollupJobs.create.stepHistogram.fieldsChooser.label"
+              defaultMessage="Select histogram fields"
+            />
+          )}
+          fields={unselectedHistogramFields}
+          onSelectField={this.onSelectField}
+        />
+
+        {this.renderInterval()}
+
+        {this.renderErrors()}
+      </Fragment>
+    );
+  }
+
+  renderInterval() {
+    const {
+      fields,
+      onFieldsChange,
+      areStepErrorsVisible,
+      fieldErrors,
+    } = this.props;
+
+    const {
+      histogram,
+      histogramInterval,
+    } = fields;
+
+    const {
+      histogramInterval: errorHistogramInterval,
+    } = fieldErrors;
+
+    if (!histogram.length) {
+      return null;
+    }
+
+    return (
+      <Fragment>
+        <EuiSpacer size="l" />
+
+        <EuiTitle size="s">
+          <h4>
+            <FormattedMessage
+              id="xpack.rollupJobs.create.stepHistogram.sectionHistogramInterval.title"
+              defaultMessage="Histogram interval"
+            />
+          </h4>
+        </EuiTitle>
+
+        <EuiText>
+          <FormattedMessage
+            id="xpack.rollupJobs.create.stepHistogram.sectionHistogramInterval.description"
```

Comment:
should be `xpack.rollupJobs.create.stepHistogram.sectionHistogramIntervalDescription`